### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by mcroni** [codewaseem/capacitor-filepicker-plugin#1]
Kindly provide some writeup on how to use this plugin. Thanks

**Comment by habeeba76429** [comment]
Hi @mcroni,

I'm a bit busy right now. I will add the documentation as soon as possible. But till then refer the following piece of code, this was how I have used it in my project
`
 handleNativeFileSelect = async () => {
        if(Capacitor && Capacitor.isNative && Capacitor.Plugins.FilePicker) {
            let fileInfo = await Capacitor.Plugins.FilePicker.pick();
            
            let fileBlob = await fetch(window.Ionic.WebView.convertFileSrc(fileInfo.uri)).then(r => r.blob());
            let ext = mime.getExtension(fileBlob.type);
            let file = new File([fileBlob],`file.${ext}`);
            this.setState(() => {
               return {
                   file,
                   uri: fileInfo.uri
               }
            });

        } else {
            console.log("Filepicker plugin not installed");
        }
    }
`

**Comment by mobius3** [comment]
Care to share the fetch() implementation? what is it?

**Comment by codewaseem** [comment]
@mobius3 The `fetch` API is the [standard Web API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API).

Here I'm using it to `fetch` the actual file picked from the phone/sd-card storage. The `filePickerPlugin` only provides the URI of the file, not the file itself.

To get the actual file data , You have to convert the `android://uri/to/file` URI to normal URI (`file:///uri/to/file`) using `window.Ionic.WebView.convertFileSrc(fileInfo.uri)`. The following three lines of code does that.

```
let fileBlob = await fetch(window.Ionic.WebView.convertFileSrc(fileInfo.uri)).then(r => r.blob());
let ext = mime.getExtension(fileBlob.type);
let file = new File([fileBlob],`file.${ext}`);
```

**Comment by codewaseem** [comment]
This commit number e7de25d, closes #1

**Comment by mobius3** [comment]
Thanks!
